DEVGUIDE.md

```diff
@@ -196,6 +196,17 @@ or
 > **Note**
 > Please note, that by default, **Release** version of IL baseline tests will be running in CI, so when updating baseline (.bsl) files, make sure to add `-c Release` flag to the build command.
 
+
+### Updating FCS surface area baselines
+
+```bash
+$env:TEST_UPDATE_BSL=1
+dotnet test tests/FSharp.Compiler.Service.Tests/FSharp.Compiler.Service.Tests.fsproj --filter "SurfaceAreaTest" /p:BUILDING_USING_DOTNET=true
```

Comment:
This won't work if building with build scripts. The dotnet building flag is not officially supported with arcade builds, and will yield cryptic errors.

This should instead explain how to update surface area with build scripts and separately with dotnet test, same as it is done with builds.